esmvalcore/preprocessor/_io.py

```diff
@@ -187,17 +188,110 @@ def load(file, callback=None, ignore_warnings=None):
     return raw_cubes
 
 
-def _by_two_concatenation(cubes):
-    """Perform a by-2 concatenation to avoid gaps."""
-    concatenated = iris.cube.CubeList(cubes).concatenate()
-    if len(concatenated) == 1:
-        return concatenated[0]
+def _concatenate_cubes(cubes, check_level):
+    """Concatenate cubes according to the check_level."""
+    kwargs = {
+        'check_aux_coords': True,
+        'check_cell_measures': True,
+        'check_ancils': True,
+        'check_derived_coords': True
+    }
 
-    concatenated = _concatenate_overlapping_cubes(concatenated)
-    if len(concatenated) == 2:
-        _get_concatenation_error(concatenated)
-    else:
-        return concatenated[0]
+    if check_level > CheckLevels.DEFAULT:
```

Comment:
isn't check_level a string? What's this semantic comparison do?

Reply:
It should be a CheckLevels instance, which sets a number to each level;

https://github.com/ESMValGroup/ESMValCore/blob/3222d28a8587e7ce3f5871522c035079b645dd5b/esmvalcore/cmor/check.py#L18

Reply:
gotcha, thanks @sloosvel